app/assets/stylesheets/homepage.css.scss

```diff
@@ -38,8 +38,13 @@
     padding: 9px 12px 0 12px;
 }
 
+.nav-bar-link, .nav-bar-link:visited, .nav-bar-link:link, #logo, #logo:visited, #logo:link {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line